### ISSUE TITLE
Forbid to have too many compares in one expression

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#639]
# Rule request

## Thesis

We need to forbid expressions with too many `ast.Compare` parts:

```python
x > y > z > a > b > c
```

I would say, that we need to allow `3` or `4` elements max.
It is also possible to keep `==` with `4` elements. And other compare elements can have only `3` parts: `z > x > y`.

## Reasoning

It is hard to read elements with multiple compare cases.

**Comment by kxepal** [comment]
Does `0 < x <= 1 and 0 < y <= 1` violates this rule idea?

**Comment by sobolevn** [comment]
Nope, `<` can be used with `<=` and `>` with `>=`

**Comment by kxepal** [comment]
So it's just about > / < comparison chain length?

**Comment by sobolevn** [comment]
Yep, that's exactly it. 

P.S. I have meant to close https://github.com/wemake-services/wemake-python-styleguide/issues/614 instead
